### PR TITLE
makes the hos gun things again

### PR DESCRIPTION
**Pull request by triplezeta**
and you're stuff

## About The Pull Request

resprites the hosgun and laser gun

caplaserbig
funygun

also rebalances the HOSgun slightly. Taser mode has been readded (5 shots until empty), ion mode's cost has been reduced (10 shots until empty)

~~NOTE: DO NOT MERGE UNTIL HOS GUN CHARGE OVERLAY IS FIXED~~ ok guys mark fixed it

## Why It's Good For The Game

???????????????????

## Changelog
:cl:
balance: The Head of Security's X-01 energy gun has had its ion bolt lens reshaped slightly, allowing for far more efficiency in projecting ion bolts. It is now capable of firing up to 10 ion bolts before needing to recharge.
balance: The Head of Security's energy gun has additionally had a fourth lens added to its optics array, allowing for the firing of taser electrodes. 
imageadd: An assistant offered the Head of Security and Captain a "makeover" for their "crusty, soul covered guns"
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
😳

**Comment by TheNeoGamer42** [comment]
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh

**Comment by triplezeta** [comment]
@TheNeoGamer42 stuff

**Comment by TheNeoGamer42** [comment]
stuff noises

File: code/modules/projectiles/guns/energy/energy_gun.dm [hunk 1]
```diff
@@ -58,7 +58,7 @@
 	cell_type = /obj/item/stock_parts/cell/gun/upgraded
 	icon_state = "hoslaser"
 	force = 10
-	ammo_type = list(/obj/item/ammo_casing/energy/disabler/hos, /obj/item/ammo_casing/energy/laser/hos, /obj/item/ammo_casing/energy/ion/hos)
+	ammo_type = list(/obj/item/ammo_casing/energy/disabler/hos, /obj/item/ammo_casing/energy/laser/hos, /obj/item/ammo_casing/energy/ion/hos, /obj/item/ammo_casing/energy/electrode/hos)
 	automatic = 1
 	fire_rate = 3
 	ammo_x_offset = 4
```
Comment (MarkSuckerberg) [inline code/modules/projectiles/guns/energy/energy_gun.dm:64]:
```suggestion
	shaded_charge = TRUE
```

**Comment by SynnGraffkin** [comment]
Please appropriately name your PR

**Comment by triplezeta** [comment]
im stuff

**Pull request merged**